tests/integration/cairo1_compiler_bindings/library_functions_test/invoke_test.cairo

```diff
@@ -0,0 +1,22 @@
+use array::ArrayTrait;
+
+#[test]
+fn test_invoke() {
+   let mut arr = ArrayTrait::new();
+   arr.append(101);
```

Comment:
Can we create a second test with an empty array?

Reply:
[62fa1d4](https://github.com/software-mansion/protostar/pull/1527/commits/62fa1d411ae0d6c53089b78c8bee7a61e654a8f5)